### ISSUE TITLE
Export/import features

### ISSUE THREAD

**Issue by er0ckx** [AlexGilleran/HIITMe#3]
I really like this HIIT app.  However, it's really hard to transfer my programs from an old phone to a new one.  Basically I have to re-enter everything by hand.  I'd really appreciate an export to phone (local storage, dropbox, etc.) to back up my programs and allow me to import into a new phone.

Thanks,
Eric

**Comment by tvass** [comment]
Good idea. Also, i did NOT check if it was an internal db or files but it could be nice to be able to choose storage path to be able to point to a synced folder (like nextcloud or any cloud drive).

Also in this case, we could share trainings it w/ friends.
